docs/guides/culture.html.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cultural Changes
+subtitle: Plugin creation
+layout: guide
+order: 0
+---
+
+### Introducing Danger
+
+It can be easy to try and jump straight from no Dangerfile to a 200 line complex set of cultural rules. We'd advise against first coming up with a really long checklist of rules for Danger, and then trying to introduce them all at-once. The entire team may have agreed on the changes, but slower gradual integration has worked better for teams new to working with Danger.
```

Comment:
Building on your sentence @dangermcshane:

We'd advise against introducing a long list of rules for Danger all at once. In our experience, gradual integration works better.